bebot/src/CMakeLists.txt

```diff
@@ -0,0 +1,5 @@
+add_library(bebot bebot.cpp ../include/bebot/bebot.hpp)
```

Comment:
split CMake functoin calls over multiple files.
```
add_library(bebot 
  bebot.cpp
  othercpp.cpp
  etc.cpp
)
```
Headers shouldn't be part of your add_library call, it should be part of an include_directories() call